eventtracking/backends/tests/test_mongodb.py

```diff
@@ -47,8 +48,14 @@ def test_authentication_settings(self):
         backend = MongoBackend(user=sentinel.user, password=sentinel.password)
         backend.database.authenticate.assert_called_once_with(sentinel.user, sentinel.password)
 
-    def test_mongo_insertion_error(self):
+    def test_mongo_pymongo_insertion_error(self):
         self.backend.collection.insert.side_effect = PyMongoError
 
         self.backend.send({'test': 1})
         # Ensure this error is caught
+
+    def test_mongo_bson_insertion_error(self):
+        self.backend.collection.insert.side_effect = BSONError
+
+        self.backend.send({'test': 1})
```

Comment:
Are we using a real backend at this point?  If so, it would be nice to add a test that really causes the BSONError (uses an unsupported character in the key).  The test could verify that the event was not added and that the error is suppressed.